code/game/atoms_movable.dm

```diff
@@ -26,7 +26,7 @@
 	var/datum/forced_movement/force_moving = null	//handled soley by forced_movement.dm
 	var/floating = FALSE
 
-/atom/movable/SDQL_update(const/var_name, new_value)
+/atom/movable/vv_edit_var(var_name, var_value)
 	if(var_name == "step_x" || var_name == "step_y" || var_name == "step_size" || var_name == "bound_x" || var_name == "bound_y" || var_name == "bound_width" || var_name == "bound_height")

```

Comment:
and add a comment above every single vv_edit_var proc and override specifying that this should be the way its done.